types/src/transaction/mod.rs

```diff
@@ -71,15 +71,16 @@ pub struct RawTransaction {
 
     gas_currency_code: String,
 
-    // Expiration time for this transaction.  If storage is queried and
+    // Expiration timestamp for this transaction. timestamp is represented
+    // as duration in seconds from Unix Epoch. If storage is queried and
     // the time returned is greater than or equal to this time and this
     // transaction has not been included, you can be certain that it will
     // never be included.
     // A transaction that doesn't expire is represented by a very large value like
     // u64::max_value().
     #[serde(serialize_with = "serialize_duration")]
     #[serde(deserialize_with = "deserialize_duration")]
-    expiration_time: Duration,
+    expiration_timestamp: Duration,
```

Comment:
Is this a breaking change (and add the breaking label)?

Reply:
I don't think so. It's just a naming change that shouldn't affect lcs result.